arachne/server/SubgraphIsomorphism.chpl

```diff
@@ -117,26 +342,31 @@ module SubgraphIsomorphism {
                     }
                     else {
                         // Recursively call the function for the next vertex
-                        var subIsoList = ullmannSubgraphIsomorphism11Helper(G, H, v+1, visited, mapping, graphDegree);
+                        var subIsoList = ullmannSubgraphIsomorphism11Helper(G, H, v+1, visited, mapping, graphDegree, st);
+                        //writeln("subIsoList= ",subIsoList );
                         if (subIsoList.size > 0) {
                             // Add isomorphisms found in the sub-branch to the current isomorphism list
                             for isoMapping in subIsoList {
                                 localIsoList.pushBack(isoMapping);
+                                writeln("pushBack of ",isoMapping );
                             }
                         }
                     }
                 }
-                writeln("$$$$$$ WE GET HERE 5");
                 // Backtrack: unvisit the vertex and remove it from the mapping
                 visited[i] = false;
                 mapping[v] = -1;
             }
         }
+        writeln("localIsoList= ", localIsoList);
```

Comment:
Remove this line or else the server will take up a lot of time writing this out and mess up performance metrics.